frontend/src/context/layers/composite_data.ts

```diff
@@ -41,25 +46,22 @@ export const fetchCompositeLayerData: LazyLoader<CompositeLayerProps> =
       })),
     };
 
-    // eslint-disable-next-line no-console
-    console.log('Request config used for Qmulti:', {
-      body,
-    });
     try {
-      const response = await fetchWithTimeout(
-        baseUrl,
-        dispatch,
-        {
-          body: JSON.stringify(body),
-          method: 'POST',
-          timeout: 600000, // 10min
-          headers: {
-            Accept: 'application/json',
-            'Content-Type': 'application/json',
-          },
+      const response = await fetchWithTimeout(baseUrl, dispatch, {
+        body: JSON.stringify(body),
+        method: 'POST',
+        timeout: 600000, // 10min
+        headers: {
+          Accept: 'application/json',
+          'Content-Type': 'application/json',
         },
-        `Request failed for fetching boundary layer data at ${baseUrl}`,
-      );
+      });
+
+      // eslint-disable-next-line no-console
+      console.log('Request config used for Qmulti:', {
```

Comment:
We can probably get rid of this now? It's always in the network requests if we need it

Reply:
Yes, definitely